src/UserFunctions.java

```diff
@@ -0,0 +1,129 @@
+public class UserFunctions {
+
+    // This needs the Puzzle class
+    // private Puzzle puzzle;
+
+    private List<String> display;
+
+    // public UserFunctions(Puzzle p) {
+    //  puzzle = p;
+    //  display = new List<>();
+    //}
+
+    public UserFunctions() {
+        //_puzzle = new Puzzle();
+        display = new List<>();
+    }
+
+    private String exitCommand = "exit";
+    private String foundCommand = "found";
+    private String helpCommand = "help";
+    private String loadCommand = "load";
+    private String newCommand = "new";
+    private String saveCommand = "save";
+    private String showCommand = "show";
+    private String shuffleCommand = "shuffle";
+
+    /* 
+     * Processes the given input and performs the appropriate function.
+     * Parameters:
+     *   input: The command being processed.
+     * Returns:
+     *   If the quit command was given, returns false; otherwise, returns true.
+     */
```

Comment:
Consider using JavaDoc style comments for function documentation

```suggestion
    /** 
     * Processes the given input and performs the appropriate function.
     *
     * @param input The command being processed.
     * @return If the quit command was given, returns false; otherwise, returns true.
     */
```